.github/workflows/release.yaml

```diff
@@ -0,0 +1,66 @@
+---
+name: Release
+
+run-name: release ${{ inputs.tag }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to be released'
+        required: true
+
+permissions:
+  packages: write
+  id-token: write
+  contents: read
+
+jobs:
+  check:
+    name: preliminary-checks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Ensure version is properly set
+        run: |
+          test "$(yarn -s run get-version)" = "${{ inputs.tag }}"
+      - name: Give user instruction
+        if: failure()
+        run: echo "::error file=package.json,line=6::The tag ${{ inputs.tag }} must match the version"
+  publish:
+    runs-on: ubuntu-latest
+    needs: check
+    steps:
+      - uses: actions/checkout@v4
+      # Setup .npmrc file to publish to GitHub Packages
+      - uses: actions/setup-node@v4
+        with:
+          cache: yarn
+          node-version: '16'
+          registry-url: 'https://npm.pkg.github.com'
+      - run: yarn install --frozen-lockfile
+      - name: Publish to GitHub Packages
+        run: npm publish --provenance
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: configure registry to npmjs.org
+        uses: actions/setup-node@v4
+        with:
+          node-version: '16'
```

Comment:
maybe remove this node version here, only change the registry

Reply:
I reopon, looking at https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#publish-to-npmjs-and-gpr-with-yarn they don't provide node version in the second one

Reply:
ok thanks, will follow then.